tests/e2e/main.go

```diff
@@ -280,6 +286,7 @@ func getTestCases(selectedPredefinedTests, selectedTestFiles TestSet, providerVe
 			"consumer-double-downtime", "partial-set-security-opt-in", "partial-set-security-top-n",
 			"partial-set-security-validator-set-cap", "partial-set-security-validators-power-cap",
 			"partial-set-security-validators-allowlisted", "partial-set-security-validators-denylisted",
+			"partial-set-security-validators-allowlisted", "too-many-validators",
```

Comment:
1. Should this have been `active-set-changes` instead of `too-many-validators`?
2. Does this reintroduce `partial-set-security-validators-allowlisted`?